sdk/include/ndk/setypes.h

```diff
@@ -108,6 +118,16 @@ typedef struct _TOKEN_ACCESS_INFORMATION
      SE_GROUP_INTEGRITY                                 |   \
      SE_GROUP_INTEGRITY_ENABLED)
 
+//
+// Privilege token filtering flags
+//
+#define DISABLE_MAX_PRIVILEGE 0x1
+#define SANDBOX_INERT         0x2
+#if (NTDDI_VERSION >= NTDDI_LONGHORN)
+#define LUA_TOKEN             0x4
+#define WRITE_RESTRICTED      0x8
+#endif
+
```

Comment:
Why duplicated? If you want to have them both available for usermode and kernelmode from the NDK itself, just keep the definition that is outside of the `NTOS_MODE_USER` define'd region.